### PR TITLE
refactor(rxEnvironment): stop using rxEnvironment.get()

### PR DESCRIPTION
**Pull request by cathysiller**
JIRA: https://jira.rax.io/browse/FRMW-1253

### LGTMs
- [x] Dev LGTM
- [x] Dev+Vidyo LGTM

### CODE REVIEW

File: src/utilities/rxUrlUtils/scripts/rxUrlUtils.js [hunk 1]
```diff
@@ -106,6 +106,7 @@ angular.module('encore.ui.utilities')
         }
 
         // run the href through rxEnvironmentUrl in case it's defined as such
+        // remove in EncoreUI 4.0.0
         url = rxEnvironmentUrlFilter(url);
```
Comment (CITguy) [inline src/utilities/rxUrlUtils/scripts/rxUrlUtils.js]: As it currently stands, the `rxEnvironmentUrl` filter is just passing through the `url` value. However, removing this line causes errors in `rxAppRoutes`. Given that `rxAppRoutes` is slated for removal in 4.0.0, we'll revisit removing this line in 4.0.0 updates.

**Comment by CITguy** [comment]
We'll wait for @parlarjb to give the 👍  before merging this one.

File: src/utilities/rxFavicon/scripts/rxFavicon.js [hunk 2]
```diff
@@ -38,23 +38,18 @@ angular.module('encore.ui.utilities')
             favicons.prod = el.attr('href');
             favicons.staging = favicons.staging || favicons.prod;
             favicons.local = favicons.local || favicons.staging;
+            
+            var currentEnv;
+            if (rxEnvironment.isLocal()) {
+                currentEnv = 'local';
+            } else if (rxEnvironment.isUnifiedPreProd()) {
+                currentEnv = 'staging';
```
Comment (parlarjb) [inline src/utilities/rxFavicon/scripts/rxFavicon.js]: Unfortunately this logic isn't correct, as `isUnifiedPreProd()` will return true for staging _and_ preprod.

I'd do this:

```
currentEnv = 'staging';
if (rxEnvironment.isLocal()) {
    currentEnv = 'local';
} else if (rxEnvironemnt.isPreProd() || rxEnvironment.isUnifiedProd() {
    currentEnv = 'prod'
}
```

or something along those lines. Our preprod behaviour is actually undefined here, so I'm not sure what the right thing is to do

Comment (cathysiller) [inline src/utilities/rxFavicon/scripts/rxFavicon.js]: updated

**Comment by parlarjb** [comment]
LGTM

**Pull request merged**